Tools/clinic/clinic.py

```diff
@@ -1944,9 +1970,16 @@ def render_function(
         full_name = f.full_name
         template_dict = {'full_name': full_name}
         template_dict['name'] = f.displayname
-        if f.kind is GETTER:
-            template_dict['getter_name'] = f.c_basename.upper() + "_GETTERDEF"
-            template_dict['c_basename'] = f.c_basename + "_get"
+        if f.kind in (GETTER, SETTER):
+            template_dict['getsetter_name'] = f.c_basename.upper()
+            template_dict['getset_basename'] = f.c_basename
+            if f.kind is GETTER:
+                template_dict['c_basename'] = f.c_basename + "_get"
+            elif f.kind is SETTER:
+                template_dict['c_basename'] = f.c_basename + "_set"
+                # TODO: Improve handling pararms better way.
+                data.impl_parameters.append("PyObject *value")
```

Comment:
@erlend-aasland @AlexWaygood 
If you want to propose a better way for this, please let me know.

Reply:
Yeah, if the current implicit declaration is acceptable, I am fine.
But we may consider refactoring them at some moment, even if we declare it implicitly.